test/frontend_test.js

```diff
@@ -86,7 +86,7 @@ describe('Automerge.Frontend', () => {
     it('should delete keys in maps', () => {
       const actor = uuid()
       const [doc1, change1] = Frontend.change(Frontend.init(actor), doc => { doc.magpies = 2; doc.sparrows = 15 })
-      const [doc2, change2] = Frontend.change(doc1, doc => delete doc['magpies'])
+      const [doc2, change2] = Frontend.change(doc1, doc => delete doc['magpies']) // eslint-disable-line
```

Comment:
eslint encourages consistent use of dot notation where the property is precomputed, but these uses seemed deliberate. Thoughts on that, @ept?